bubble_sort.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+def bubble_sort(array)
+  not_sorted = true, n = array.length - 1
+  sorting(not_sorted, array, n)
+  array
+end
+
+def sorting(not_sorted, array, index)
+  while not_sorted
+    not_sorted = false
+    (0...index).each do |i|
+      next unless array[i] > array[i + 1]
+
+      array[i], array[i + 1] = array[i + 1], array[i]
+      not_sorted = true
+    end
+    index -= 1
+  end
+end
+
```

Comment:
**bubble_sort_by** is missing in this file

Reply:
my partner wants to work alone , thats why i am also working alone. my partner dont' wants to work with me.